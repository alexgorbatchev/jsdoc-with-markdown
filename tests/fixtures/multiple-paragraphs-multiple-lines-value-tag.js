/**
 * This is a description.
 * And this is more of it.
 *
 * @class Foo Bar
 *
 * This is a second paragraph
 * with two lines.
 */
