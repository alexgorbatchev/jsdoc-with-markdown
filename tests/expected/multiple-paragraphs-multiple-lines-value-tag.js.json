[
  {
    "type": "paragraph",
    "value": "This is a description. And this is more of it."
  },
  {
    "type": "tag",
    "key": "class",
    "value": "Foo Bar"
  },
  {
    "type": "paragraph",
    "value": "This is a second paragraph with two lines."
  }
]
